tailwind.config.js

```diff
@@ -1,8 +1,16 @@
+const defaultTheme = require('tailwindcss/defaultTheme');
+const colors = require('tailwindcss/colors');
+
 /* eslint-disable global-require */
 module.exports = {
   purge: false,
   darkMode: 'media', // or 'media' or 'class'
   theme: {
+    colors: {
+      ...defaultTheme.colors,
+      ...colors,
+      gray: colors.trueGray, // colors.coolGray, colors.blueGray, etc.
+    },
```

Comment:
This is where you can define colors. I'm not 100% set on using `trueGray` palette though, so feel free to experiment!

If you'd like to define a custom one, it would look something like this:

```js
gray: {
    '50': '#f8f8f8', 
    '100': '#f1f1f1', 
    '200': '#dbdbdb', 
    '300': '#c6c6c6', 
    '400': '#9b9b9b', 
    '500': '#707070', 
    '600': '#656565', 
    '700': '#545454', 
    '800': '#434343', 
    '900': '#373737'
}
```

I used one of many [TW palette generators](https://javisperez.github.io/tailwindcolorshades/?dove-gray=707070) and [here's an overview of default palette](https://tailwindcss.com/docs/customizing-colors#color-palette-reference)  that comes with tailwind for reference.

Reply:
Oh nice! I knew there must be something I didn't know how to google for